sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_auth_events_v1/query.sql

```diff
@@ -22,4 +24,4 @@ WHERE
   jsonPayload.type = 'amplitudeEvent'
   AND jsonPayload.fields.event_type IS NOT NULL
   AND jsonPayload.fields.user_id IS NOT NULL
-  AND _TABLE_SUFFIX = FORMAT_DATE('%y%m%d', @submission_date)
+  AND PARSE_DATE('%y%m%d', _TABLE_SUFFIX) = DATE(@submission_date)
```

Comment:
Let's remove this change and keep it as it was
```suggestion
  AND _TABLE_SUFFIX = FORMAT_DATE('%y%m%d', @submission_date)
```